### ISSUE TITLE
Centralise laser cutter troubleshooting guide

### ISSUE THREAD

**Issue by johnmckerrell** [DoESLiverpool/somebody-should#1599]
Sophia has a troubleshooting guide but Gerald does not, it's probably worth centralising this given they both share some (but not all) issues. Should add #1598 solution to the list when doing so, also the issues mentioned in the last comment on there.

**Comment by arthurrowland** [comment]
I have started the page, basically copying the info across from Sheila and outlining the sections of the page. Be good if someone could sense check it, and of course continue adding troubleshoots.

https://github.com/DoESLiverpool/somebody-should/wiki/LaserCutterTroubleshooting

The notes from #1598 copied across for reference and monitoring progress:

- [ ] I need more material!
- [ ] The image I want to etch won't load
- [ ] It won't send the file from Lasercut to the machine
- [ ] It's not cutting the same across the bed (particularly on the right hand side)
- [ ] The lines are wobbly
- [ ] The laser head stopped moving
- [ ] The laser head stopped firing
- [ ] The light went out
- [ ] There's a second cut line or 'shadow'

**Comment by amcewen** [comment]
It feels like the laser-cutting area of the wiki could do with a tidy-up in general.  Maybe splitting the "using the software" off into separate pages for Lightburn and LaserCut and pointing to those from the equipment pages?

I'm here because someone was trying to match the [instructions for Gerald on the wiki](https://github.com/DoESLiverpool/somebody-should/wiki/LaserCutterGerald) with the software installed; and as you'd expect, getting *more* rather than *less* confused.

I've added a warning to Gerald's page that the instructions are out-of-date, but it'd be nice if they could be updated.

(Tagging @DoESLiverpool/laser-maintenance as it feels like they should be involved in reworking things)

**Comment by huffeec** [comment]
This feels relevant here rather than a separate issue.

A user today opened Lightburn to see a dialogue box for connecting the device, and was thrown by the lack of Gerald option. We selected the Smoothieware option and the were able to get Gerald working but it might be useful to add this to a troubleshooting guide. It's not something I had seen before and it isn't clear what would cause it.

**Comment by DaveBenji** [comment]
I presume Gerald is working now?

**Comment by huffeec** [comment]
Yes, it worked fine. Just made a record of the confusion caused by the software.
